src/spec-node/featuresCLI/publish.ts

```diff
@@ -90,11 +90,45 @@ async function featuresPublish({
             process.exit(1);
         }
 
-        const thisResult = (publishResult?.digest && publishResult?.publishedVersions.length > 0) ? {
+        const ispublished = (publishResult?.digest && publishResult?.publishedVersions.length > 0);
+        let thisResult = ispublished ? {
             ...publishResult,
             version: f.version,
```

Comment:
I think I would rather have `publishedLegacyIds` always a part of the returned object (an empty array to start, which the loop below can append to if necessary).

That way, we either always get:
1. An object with all properties (if something actually happened, ie: We published tags)
2. A completely empty object (if nothing happened)

Reply:
Renaming a Feature would happen rarely, so I was thinking that we shouldn't always add this object. It could confuse people as there would be very few people who would benefit from it.

Also, this way we could have a clear publishResult, and if something goes wrong, we could always look at the logs 😅

Reply:
for eg. if you look at a sample 

![image](https://user-images.githubusercontent.com/24955913/209198105-ed04146a-1242-4e4e-ba4d-852352490744.png)

There is already so much info, but this is currently very helpful. However, if every item had `publishedLegacyIds: {}` would it become a bit extra? 🤔 

These were my thoughts when I made this code change, but happy to modify if we think otherwise.